### PR TITLE
Allow selection of release/staging/devel images

### PR DESCRIPTION
**Pull request by kiall**
The STAGING variable, and it's values, match those used in caasp-devenv

### CODE REVIEW

**Comment by kiall** [comment]
Marking as WIP, this seems to download the image over + over

**Comment by kiall** [comment]
The "download over + over" thing was only on Devel images, as the download.suse.de folder has duplicate copies of the image. Updating the wget lines slightly to avoid it matching these files.

File: contrib/libvirt/k8s-libvirt.sh [hunk 1]
```diff
@@ -61,10 +59,19 @@ elif [ "$prefix" = "" ]; then
...
+    elif [ "$STAGING" == "staging_a" ]; then
+        CAASP_IMAGE_BASE_URL="http://download.suse.de/ibs/SUSE:/SLE-12-SP2:/Update:/Products:/CASP10:/Staging:/A/images/"
+    elif [ "$STAGING" == "staging_b" ]; then
+        CAASP_IMAGE_BASE_URL="http://download.suse.de/ibs/SUSE:/SLE-12-SP2:/Update:/Products:/CASP10:/Staging:/B/images/"
+    elif [ "$STAGING" == "devel" ]; then
+        CAASP_IMAGE_BASE_URL="http://download.suse.de/ibs/Devel:/CASP:/1.0:/ControllerNode/images/"
+    else
+        echo "[+] ERROR: Unknown CaaSP image: ${STAGING}"
+        exit 1
+    fi
```
Comment (ereslibre) [inline contrib/libvirt/k8s-libvirt.sh]: I don't mean to be picky, but would a `case` statement fit better here? Just for readability.

**Comment by kiall** [comment]
Cool, I actually want to extract all the download logic into something separate anyway, this file is getting huge :)

**Pull request merged**